cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -289,11 +289,15 @@ def finish_flow_cell(
     def finish_all_flow_cells(self, bcl_converter: str) -> None:
         """Loop over all flow cells and post process those that need it."""
         for flow_cell_dir in self.get_all_demultiplexed_flow_cell_dirs():
-            self.finish_flow_cell(
-                bcl_converter=bcl_converter,
-                flow_cell_name=flow_cell_dir.name,
-                flow_cell_path=flow_cell_dir,
-            )
+            try:
+                self.finish_flow_cell(
+                    bcl_converter=bcl_converter,
+                    flow_cell_name=flow_cell_dir.name,
+                    flow_cell_path=flow_cell_dir,
+                )
+            except Exception as error:
```

Comment:
Yes, I agree that in general, we should specify the exception type otherwise we could overlook some errors. However in the CLI commands that iterate over all flow cells (i.e. the ones that have the prefix `-all`), for any error in the flow cell, production will be hated as the process will exit and not start execution on potentially correct flow cells.